src/main/java/uk/gov/hmcts/dts/fact/util/AddressType.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.dts.fact.util;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+public enum AddressType {
+    VISIT_US("Visit us", true),
+    WRITE_TO_US("Write to us", false),
+    VISIT_OR_CONTACT_US("Visit or contact us", true);
+
+    private static final Map<String, AddressType> LOOKUP = new ConcurrentHashMap<>();
+
+    private final String name;
+    private final boolean isCourtAddress;
+
+    static {
```

Comment:
super minor, can be private?

Reply:
This is a static initializer not a class so it can't be private :) The method values() is not our method, it is provided by Java for enum. So AddressType.values() with return all the enum values for that enum class. It is not necessary to use AddressType.values() (just value() is sufficient) because we are in that class scope.